packages/flutter/lib/src/painting/decoration_image.dart

```diff
@@ -142,6 +146,35 @@ class DecorationImage {
   /// calculated by multiplying [scale] with `scale` of the given [ImageProvider].
   final double scale;
 
+  /// If non-null, the value is multiplied with the opacity of each image
+  /// pixel before painting onto the canvas.
+  ///
+  /// This is more efficient than using [Opacity] or [FadeTransition] to
+  /// change the opacity of an image.
+  final double opacity;
+
+  /// Used to set the filterQuality of the image
```

Comment:
nit: `image.`

(missing period)

Reply:
Also, I believe the style guide calls for a newline between this and following docstring content, as your other docstrings do :)